examples/pylab_examples/axes_demo.py

```diff
@@ -1,32 +1,34 @@
 #!/usr/bin/env python
 
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy as sp
```

Comment:
We shouldn't use Scipy. As is it not installed on our Travis machine and we try avoiding to depend on it. 

You can get randn from numpy.random

```
import numpy as np
np.random.randn
```